envs/oracledb/12cr1/groups/12cr1.yaml

```diff
@@ -0,0 +1,319 @@
+name: "oracledb::12cr1"
+description: "DB Oracle 12c R1"
+
+# Define the LXD Remote to use and where
+# create the environment.
+connection: "local"
+# Define the list of LXD Profile to use
+# for create the containers
+common_profiles:
+  - oracledb
+  - privileged
+  - memory4g
+
+# Create the environment container as ephemeral or not.
+ephemeral: true
+
+nodes:
+
+  - name: oracle12cr1
+    image_source: "oracle/7"
+    image_remote_server: "images"
+
+    # Specify directory where build path of templates.
+    # Directory could be an absolute path or a relative path based
+    # on env base dir.
+    source_dir: "12cr1/templates/"
+
+    # List of commands executed just after the creation of the
+    # container.
+    hooks:
+      - event: pre-node-sync
+        flags:
+          - packets-install
+        commands:
+          # general packets
+          - yum install -y $(echo ${general_packages} | jq '.[]' -r)
+          # common packages specific for oracle
+          - yum install -y $(echo ${common_packages_ora} | jq '.[]' -r)
+          # specific packages for oracle database
+          - yum install -y $(echo ${specific_packages_ora} | jq '.[]' -r)
+      - event: pre-node-sync
+        flags:
+          - oradbinstance
+          - oraswdbinstall
+          - oralistener
+          - generalpost
+          - oradbpluggables
+        commands:
+          - rm -r /tmp/*.rsp || echo "NO FILE FOUND"
+          - rm -r /tmp/*.service || echo "NO FILE FOUND"
+          - rm -r /tmp/.profile* || echo "NO FILE FOUND"
+      - event: pre-node-sync
+        flags:
+          - oradbinstance
+          - oralistener
+          - oraswdbinstall
+          - oradbpluggables
+          - general
+        commands:
+          - hostname | xargs | tr '\n' ' '
+        out2var: "hostnamevar"
+      - event: pre-node-sync
+        flags:
+          - common
+        commands:
+          - >
+            echo "################" &&
+            echo "COMMON OPERATION" &&
+            echo "################"
+          - test -d ${oracle_base}"/"${db_version}"/" && (rm -rf /u01; echo "empty oracle dirs"; exit 0) || echo "oracle dirs not present"
+          - a=`yum repolist | grep epel`; test -n "$a" || yum install -y ${epel_repo} 
+          - yum install -y wget htop rlwrap sudo
+          - wget ${ol7_repo_prefix}"/"${ol7_repo}
+          - mv ${ol7_repo} /etc/yum.repos.d/          
+          - yum repolist
+          - echo "${motdfile}" > /etc/motd
+          - sed -i -e "s:HOSTNAME:${hostnamevar}:g" /etc/motd
```

Comment:
You can define multiple expressions in the same command. I mean:
```yaml
- >-
  sed -i
  -e "s:HOSTNAME:${hostnamevar}:g" 
  -e "s:DOMAINNAME:`cat /etc/resolv.conf | grep search | sed 's/search //'`:g"
   /etc/motd
```
This speedup things I think.

Reply:
added fix to compact the sed expression

Reply:
updated section networks and profiles